internal/analytics/deferred.go

```diff
@@ -22,13 +24,17 @@ type deferredData struct {
 	Dimensions map[string]string
 }
 
-const deferredCfgKey = "deferrer_analytics"
 const deferrerFileName = "deferrer"
+const deferredDataFileName = "deferred_data"
```

Comment:
I can't tell why we need both. The first one we just seem to use to look at the time it was last modified, which the second one should already provide.

Reply:
The first one is only created on the *first* deferral event, so basically the first time a deferred event is created (after the last "flush").  That way we can ensure that we flush at least once a day: https://github.com/ActiveState/cli/blob/master/internal/analytics/deferred.go#L43-L44

Reply:
With the patch that you introduced, this would never trigger, if a user runs only deferred commands, and if they run at least one of them per day.